src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockCaseAndDefaultWithTextOption.java

```diff
@@ -0,0 +1,94 @@
+/*
+EmptyBlock
+option = text
+tokens = LITERAL_DEFAULT, LITERAL_CASE, LITERAL_SWITCH
```

Comment:
More curious. Why did you add `switch` here, when you don't add it in the updated example.

Reply:
Copy paste from another file and forget to remove it. but actually, we don't need it here, removed